### PR TITLE
fix: avoid errors if Loop is not enabled

### PR DESCRIPTION
**Pull request by openoms**
Currently Loop is  not running on signet https://github.com/lightninglabs/loop/issues/522 so will needed to keep it off in https://github.com/GaloyMoney/charts/pull/1125. 

Fixing the error when deploying on signet with Loop disabled:
```
{"level":30,"time":1666863947270,"pid":1,"hostname":"api-97c4b8f8d-b4j5h","msg":"loading custom.yaml"}
/app/lib/config/process.js:88
        throw new error_1.ConfigError("Missing LND1_LOOP_TLS config");
        ^

ConfigError: Missing LND1_LOOP_TLS config
    at getLoopConfig (/app/lib/config/process.js:88:15)
    at Object.<anonymous> (/app/lib/app/swap/get-active-loopd.js:27:42)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39) {
  data: undefined
}

Node.js v18.11.0
k3d@z420:~/charts/dev$ k -n galoy-sig-galoy   logs  api-97c4b8f8d-b4j5h -f
Defaulted container "api" out of: api, wait-for-mongodb-migrate (init)
{"level":30,"time":1666863947270,"pid":1,"hostname":"api-97c4b8f8d-b4j5h","msg":"loading custom.yaml"}
/app/lib/config/process.js:88
        throw new error_1.ConfigError("Missing LND1_LOOP_TLS config");
        ^

ConfigError: Missing LND1_LOOP_TLS config
    at getLoopConfig (/app/lib/config/process.js:88:15)
    at Object.<anonymous> (/app/lib/app/swap/get-active-loopd.js:27:42)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39) {
  data: undefined
}

Node.js v18.11.0
```

### CODE REVIEW

**Comment by ntheile** [comment]
should probably also add an if statement for `cronConfig.swapEnabled` in the 

trigger.ts server - https://github.com/GaloyMoney/galoy/blob/main/src/servers/trigger.ts#L409-L412

File: src/config/process.ts [hunk 1]
```diff
@@ -103,12 +105,14 @@ export const LND_HEALTH_REFRESH_TIME_MS = parseInt(
 )
 
 export const getLoopConfig = () => {
-  if (!process.env.LND1_LOOP_TLS) throw new ConfigError("Missing LND1_LOOP_TLS config")
-  if (!process.env.LND2_LOOP_TLS) throw new ConfigError("Missing LND2_LOOP_TLS config")
-  if (!process.env.LND1_LOOP_MACAROON)
-    throw new ConfigError("Missing LND1_LOOP_MACAROON config")
-  if (!process.env.LND2_LOOP_MACAROON)
-    throw new ConfigError("Missing LND2_LOOP_MACAROON config")
+  if (getCronConfig().swapEnabled) {
```
Comment (dolcalmi) [inline src/config/process.ts:108]: this should be part of service validation

File: src/services/loopd/loop-service.ts [hunk 2]
```diff
@@ -41,6 +43,9 @@ export const LoopService = ({
   lndInstanceName,
 }: LoopdConfig): ISwapService => {
   const mac = Buffer.from(macaroon, "base64").toString("hex") as Macaroon
+  if (!tlsCert) {
```
Comment (dolcalmi) [inline src/services/loopd/loop-service.ts:46]: probably here should be the validation, also service should not throw exceptions. Please check onchain service as example

**Comment by openoms** [comment]
@dolcalmi thank you for the comments, will copy the over to https://github.com/GaloyMoney/galoy/pull/1891 which replaced this PR to be on a GaloyMoney branch

**Pull request closed without merging**